bentoml/marshal/dispatcher.py

```diff
@@ -120,6 +120,17 @@ def __init__(
         self._queue = collections.deque()  # TODO(hrmthw): maxlen
         self._sema = shared_sema if shared_sema else NonBlockSema(1)
 
+    async def cleanup(self):
+        if self._controller is not None:
+            self._controller.cancel()
+            await asyncio.sleep(0)
```

Comment:
Is `asyncio.sleep(0)` needed for connection close?

Reply:
Yep, it's unnecessary here.